android/app/build.gradle

```diff
@@ -140,6 +140,7 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
     implementation "com.facebook.react:react-native:+"  // From node_modules
+    implementation project(':react-native-svg')
```

Comment:
I believe we wanted to stay away from `react-native-svg` as it drains batteries. @tbergq could you confirm?

Reply:
I think it kind of is, but there were other requirements for this arrow: the marks on the arrow could be more numerous, and they could be square.

Reply:
And what about building arrow just from Views for line and stop squares, with `chevron-right` icon on right side? To stay away from native dependencies.

<img width="119" alt="screenshot 2018-12-07 at 09 54 56" src="https://user-images.githubusercontent.com/2660330/49637881-ec89c600-fa06-11e8-934b-f5b351305586.png">

Reply:
How about requesting this source as a png file, our see if there is some online tool converting svg -> png?

Reply:
We could do this on Figma

Reply:
@jvaclavik Right, it will be quite a lot of objects when used in list.
As I see in designs the arrow probably doesn't need to be responsive and have a fixed width.
So @tbergq solution using just images seems to me as simplest one.